src/models/src/app_state.rs

```diff
@@ -154,6 +163,52 @@ impl AppState {
             .rp_name(&rp_name);
         let webauthn = Arc::new(builder.build().expect("Invalid configuration"));
 
+        let atproto = {
+            let http_client = Arc::new(DefaultHttpClient::default());
+
+            let listen_scheme = match listen_scheme {
+                ListenScheme::Http | ListenScheme::UnixHttp => "http",
+                ListenScheme::Https | ListenScheme::HttpHttps | ListenScheme::UnixHttps => "https",
+            };
+
+            let client_metadata = AtprotoClientMetadata {
+                client_id: format!(
+                    "{listen_scheme}://{public_url}/auth/v1/atproto/client_metadata"
+                ),
+                client_uri: format!("{listen_scheme}://{public_url}"),
```

Comment:
I don't know about atproto internalals in detail, but this `client_uri` could lead to issues in scenarios, where Rauthy is hosted side-by-side with another application and only routes to `/auth/v1` are routed properly by a reverse proxy. So I guess better would be to appen `/auth/v1` just as for the token issuer.

Reply:
So, if you just host Rauthy on a particular domain, this will work like expected, because it will return an `HTTP 301` to `/auth/v1` if you access `/`. But Rauthy is also deployed in siutations, which explains the sub-domain style for the UI too, where it shares the domain with the main application.

For instance, the application is running on `example.com` but does (on purpose) not utilize any route like `/auth/*`. Then Rauthy + your app can run on the exact same domain (even though not recommended usually).
The reverse proxy then will route any traffic coming in to `/auth/*` to Rauthy and everything else to your app. They will share the same domain, but this also means, that if you define `/` as the `client_uri`, it would point to your application instead of Rauthy.